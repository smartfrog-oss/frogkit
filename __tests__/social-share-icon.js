import { mount } from 'vue-test-utils'
import ShareIcons from '@/components/social-share-icon'

const propsData = {
  url: 'https://www.smartfrog.com/de-de/ebooks/kindersicherheit/'
}

describe('Social Sharing Icons component', () => {
  const attrs = {
    facebook: '',
    twitter: '',
    gplus: ''
  }
  it('Should render component and match snapshot', () => {

    const wrapper = mount(ShareIcons, {
      propsData,
      attrs,
      attachToDocument: true
     })

    wrapper.update()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
