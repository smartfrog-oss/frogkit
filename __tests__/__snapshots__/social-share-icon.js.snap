// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social Sharing Icons component Should render component and match snapshot 1`] = `<div><a href="https://www.facebook.com/sharer/sharer.php?u=https://www.smartfrog.com/de-de/ebooks/kindersicherheit/" target="_blank" rel="noopener" class="fk-social-icon facebook"></a><a href="https://twitter.com/home?status=https://www.smartfrog.com/de-de/ebooks/kindersicherheit/" target="_blank" rel="noopener" class="fk-social-icon twitter"></a><a href="https://plus.google.com/share?url=https://www.smartfrog.com/de-de/ebooks/kindersicherheit/" target="_blank" rel="noopener" class="fk-social-icon gplus"></a></div>`;
